src/app/cube/outage-page/past-issue/past-issue.component.html

```diff
@@ -4,9 +4,9 @@
     <div>{{ issue.name | uppercase }}</div>
   </div>
   <div class="date">
-    <p>{{ issue.discovered | date:'mediumDate' }}</p>
+    <p>{{ issue.discovered | date:'medium' }}</p>
     <p>to</p>
-    <p>{{ issue.resolved | date:'mediumDate' }}</p>
+    <p>{{ issue.resolved | date:'medium' }}</p>
```

Comment:
For other reviewers: 'medium': equivalent to 'MMM d, y, h:mm:ss a' (Jun 15, 2015, 9:03:01 AM).

Reply:
yes sir